doc/api/buffer.md

```diff
@@ -538,6 +538,22 @@ added:
 Creates and returns a new `Blob` containing a subset of this `Blob` objects
 data. The original `Blob` is not altered.
 
+#### `blob.bytes()`
```

Comment:
Congrats on your first contribution! 🎉
Could you move this entry after `blob.arrayBuffer`, so that it's in alphabetical order?